### PR TITLE
Update deployment.rst

### PR DESCRIPTION
**Pull request by noniagriconomie**
Hi

Some tasks listed after this title are somehow linked to "Pre deployment" todos
For example steps A and B: that is why I propose this PR

### CODE REVIEW

File: deployment.rst [hunk 1]
```diff
@@ -101,10 +101,10 @@ specifically tailored to the requirements of Symfony.
     `Symfony plugin`_ is a plugin to ease Symfony related tasks, inspired by `Capifony`_
     (which works only with Capistrano 2).
 
-Common Post-Deployment Tasks
+Common Pre-Post-Deployment Tasks
 ----------------------------
```
Comment (OskarStark) [inline deployment.rst]: Please fix the underline

**Comment by noniagriconomie** [comment]
friendly ping @OskarStark should i target 4.4 now?
edit: i've done so according to https://github.com/carsonbot/carsonbot/issues/126

File: deployment.rst [hunk 2]
```diff
@@ -101,10 +101,10 @@ specifically tailored to the requirements of Symfony.
     `Symfony plugin`_ is a plugin to ease Symfony related tasks, inspired by `Capifony`_
     (which works only with Capistrano 2).
 
-Common Post-Deployment Tasks
-----------------------------
+Common Pre-Post-Deployment Tasks
```
Comment (OskarStark) [inline deployment.rst:104]: Please add a manual anchor to keep it working like with the headline before.

I am currently on a phone 📱 

@javiereguiluz could you please add this anchor while merging? Thanks 😊

**Comment by javiereguiluz** [comment]
Antoine, thanks for this ... it took us some time, but this is finally merged. While merging we did some tweaks and renamed the section simply as "Common Deployment Tasks", without the "Pre-Post" suffix.

**Pull request merged**